### ISSUE TITLE
Add helper function to get variable-resolved request in pre-request script

### ISSUE THREAD

**Issue by qb-adharshs** [a85/POSTMan-Chrome-Extension#5956]
When I'm sending a request with variables specified in either Params or Body tabs, it fails to get the response from the staging server that I'm testing, though it works in my local server. 

I did a little research and saw that the request body used in the pre-request section (`pm.request.body`) doesn't resolve the variables used in the Body and Params tabs, hence resulting in an MD5 validation error due to mismatch between the body used in the pre-request script and the body actually being sent with the request (when request is sent to staging, which needs authentication). 

![screenshot from 2019-02-21 09-31-23](https://user-images.githubusercontent.com/41626274/53143131-b3500380-35bc-11e9-882f-5dbf8775d41a.png)

Currently I'm using the below script to get my desired results:

For variables in Params - 
```javascript
req.url.variables.reference.order_no.value = pm.environment.get("order_no");
```

And for variables in Body - 
```javascript
req.body.raw = (req.body.raw).replace("{{order_no}}", pm.environment.get("order_no"));
```

Is there any reasoning behind this behaviour or is it a bug?
 
 - App Type - Native App
 - Postman Version - 6.7.3
 - OS: Ubuntu 16.04

**Comment by numaanashraf** [comment]
@qb-adharshs This is as per design, variables are resolved only after pre-request has finished running, as you can alter the variable values in the pre-request script. We'll consider this a feature request to expose a helper function to perform resolution within the script if needed.

**Comment by saswatds** [comment]
Duplicate of #5767
